src/rdkafka_cgrp.c

```diff
@@ -2240,9 +2242,11 @@ static int rd_kafka_cgrp_metadata_refresh(rd_kafka_cgrp_t *rkcg,
                                  rd_kafka_cgrp_handle_Metadata_op);
         rd_kafka_op_set_replyq(rko, rkcg->rkcg_ops, 0);
 
-        err = rd_kafka_metadata_request(rkcg->rkcg_rk, NULL, &topics,
-                                        rd_false /*!allow auto create */,
-                                        rd_true /*cgrp_update*/, reason, rko);
+        err = rd_kafka_metadata_request(
+            rkcg->rkcg_rk, NULL, &topics, rd_false /*!allow auto create */,
+            rd_false /*!include cluster authorized operations */,
+            rd_false /*!include topic authorized operations */,
```

Comment:
```suggestion
            rd_false /*!include cluster authorized operations */,
            rd_false /*!include topic authorized operations */,
```
```suggestion
            rd_false /*don't include cluster authorized operations */,
            rd_false /*don't include topic authorized operations */,
```

Do this at other places as well.